src/components/PaginatedTable.js

```diff
@@ -66,7 +66,8 @@ export default function PaginatedTable({columns, rows}) {
     : columns.slice(0, -1);
 
   return (
-    <Paper className="table">
+    // <Paper className={(isAdminFlow) ? "adminTable" : "employeeTable"}>
```

Comment:
Can you add a folder inside components for table and move this PaginatedTable.js and css inside it. This will helps us to maintain the structure and also we can add snapshot testing for it.